### PR TITLE
ensured automated dependency update reuses existing pull request

### PR DESCRIPTION
**Pull request by FND**
this is a bit crude and currently untested, but it _should_ do the job 🤞

### CODE REVIEW

File: bin/update-dependencies [hunk 1]
```diff
@@ -29,10 +29,30 @@ fi
...
+	method="$1"
+	shift
+	uri="$1"
+	shift
+	curl -u "$DEPENDENCIES_USER:$DEPENDENCIES_TOKEN" -X "$method" "$@" \
+		"https://api.github.com/repos/${GITHUB_REPOSITORY}${uri}"
+}
+
+# determine whether an open pull request already exists
+org=`echo "GITHUB_REPOSITORY" | sed 's#/.*##'`
```
Comment (mkhl) [inline bin/update-dependencies]: Should this be `echo "$GITHUB_REPOSITORY"`? (Note the dollar sign)

File: bin/update-dependencies [hunk 2]
```diff
@@ -29,10 +29,30 @@ fi
 git remote set-url "$remote_name" "$repo_uri"
 git push --force-with-lease "$remote_name" "$target_branch"
 
+api_request() {
+	method="$1"
+	shift
+	uri="$1"
```
Comment (mkhl) [inline bin/update-dependencies]: This isn't a URI though. Resource? Subresource? Path? (It's not the whole path but it's _a_ path.)

File: bin/update-dependencies (hunk 1 above)
Comment (mkhl) [reply bin/update-dependencies]: Or could this line be `org=${GITHUB_REPOSITORY%/*}`?

**Review by mkhl — CHANGES_REQUESTED** [review]

File: bin/update-dependencies (hunk 1 above)
Comment (FND) [reply bin/update-dependencies]: Oops, thanks for paying attention!

I don't like that kinda bashism: I can never remember the syntax, so readability is an issue from my perspective.

File: bin/update-dependencies (hunk 2 above)
Comment (FND) [reply bin/update-dependencies]: Yeah, I went back and forth on that - `path` is a reasonable suggestion though; fixed.

**Review by FND — COMMENTED** [review]
Thank you! https://github.com/faucet-pipeline/beatdown/compare/2a369dc72b3ea3f3275099d56334d2f5f671d112..508f057c32a69faed8967146ab8a3724385116b1 now includes your suggestions.

**Pull request merged**